puffin/src/experiment.rs

```diff
@@ -36,7 +36,7 @@ pub fn format_title<PB: ProtocolBehavior>(
         --{default_put_short}-{num_cores}c{without_bit_level}{without_dy_mutations}{put_use_clear}{minimizer}__\
         {title}--{hour}--{index}",
         date = date,
-        title = title.unwrap_or(GIT_REF),
+        title = title.unwrap_or(&puffin::git_ref().unwrap_or_default()),
```

Comment:
Improve `write_experiment_markdown` below with detailed information about the puts, their capabilities, etc.

Reply:
See comment above.